catalogs/catalog-jdbc-common/src/test/java/com/datastrato/gravitino/catalog/jdbc/operation/SqliteTableOperations.java

```diff
@@ -27,7 +30,12 @@ protected String generateCreateTableSql(
       String comment,
       Map<String, String> properties,
       Transform[] partitioning,
+      Distribution distribution,
       Index[] indexes) {
+    Preconditions.checkArgument(
+        null == distribution || distribution == Distributions.NONE,
+        "Currently we do not support distribution in Sqlite.");
```

Comment:
Is it that we don't support distribution or that SQLite doesn't support distribution?